### PR TITLE
Install documentation

### PR DESCRIPTION
**Pull request by ehildenb**
updates to the README following discussions about how easy it is to follow

### CODE REVIEW

**Comment by darko-marinov** [comment]
It's good that you're making these installation instructions easier to follow. Can you also add some Jenkins job to build these (e.g., using LXC containers)? Let us know if you need some help with LXC, as Yiyi and Tim have been using it for their testing.

**Comment by ehildenb** [comment]
@darko-marinov I'm not sure what you mean? Build what?

As in run the installation instructions from a fresh install?

**Comment by darko-marinov** [comment]
Yes, I meant run installation in some Jenkins job.

**Comment by dwightguth** [comment]
Jenkins: test this please

**Comment by shd101wyy** [comment]
Jenkins: test this please

**Comment by ehildenb** [comment]
@shd101wyy it's a problem in the Makefile, working on fix now

**Comment by shd101wyy** [comment]
@ehildenb I am migrating `evm-semantics-pull-request` job to run on `rvwork-2-big-jenkins-container` node. Seems like jenkins job failed again. Please let me know If I need to do anything. Thank you.

**Comment by ehildenb** [comment]
@shd101wyy I think it is an issue with your migration, because my `buildbot` server is running the tests fine. Can you take a look at the error messages? Let me know if you have questions. Seems to be something about not being able to find `prelude.k`.

**Comment by shd101wyy** [comment]
Jenkins: test this please

**Comment by shd101wyy** [comment]
@ehildenb It's working now. Something started `kserver`, which caused that error.

**Comment by ehildenb** [comment]
Jenkins: test this please

**Comment by ehildenb** [comment]
Jenkins: test this please

**Comment by ehildenb** [comment]
Jenkins: test this please

**Comment by ehildenb** [comment]
Jenkins: test this please

File: Makefile [hunk 1]
```diff
@@ -1,41 +1,57 @@
...
 
 clean:
-	rm -rf .build/java .build/ocaml .build/node .build/logs tests/proofs .build/k/make.timestamp .build/local
+	rm -rf .build/java .build/ocaml .build/node .build/logs tests/proofs .build/k/make.timestamp .build/pandoc-tangle/make.timestamp .build/local
 
-build: .build/ocaml/driver-kompiled/interpreter .build/java/driver-kompiled/timestamp
+distclean: clean
+	opam switch system
+	opam switch remove 4.03.0+k --yes || true
+	git submodule deinit --force -- ./
```
Comment (dwightguth) [inline Makefile]: does this remove untracked files in the submodules, though? if not, you will still want to do a `mvn clean` in k before you deinit

**Comment by dwightguth** [comment]
looks okay still but I have a question I'd like answered before we merge

File: Makefile (hunk 1 above)
Comment (ehildenb) [reply Makefile]: https://stackoverflow.com/questions/35940627/git-how-to-remove-untracked-folder-in-submodules

looks like it doesn't. Would you prefer `cd $(K_SUBMODULE) && mvn clean -q`, or `cd $(K_SUBMODULE) && git reset --hard`? I'm leaning towards the first, or we could do both.

Comment (dwightguth) [reply Makefile]: mvn clean would be better for a clean target.

**Comment by ehildenb** [comment]
@dwightguth changed as requested

**Pull request merged**